linter/linter_test.go

```diff
@@ -39,7 +39,7 @@ func TestNewLinter(t *testing.T) {
 func TestUnknownLinterType(t *testing.T) {
 	ruleSet := loadRulesForTest("./testdata/rules/unknown.yml", t)
 	vs := MockValueSource{}
-	_, err := NewLinter(ruleSet, vs, []string{})
+	_, err := NewLinter(ruleSet, vs, []string{}, "")
```

Comment:
Maybe I missed them when I was skimming through quickly, but I don't see any tests around passing in a value here.

I'd like to see some.